src/components/FormHelpMessage.js

```diff
@@ -35,14 +35,24 @@ const FormHelpMessage = (props) => {
         return null;
     }
 
+    const textStyles = [props.isError ? styles.formError : styles.formHelp, styles.mb0];
+
     return (
         <View style={[styles.flexRow, styles.alignItemsCenter, styles.mt2, styles.mb1, ...props.style]}>
             {props.isError && <Icon src={Expensicons.DotIndicator} fill={colors.red} />}
             <View style={[styles.flex1, styles.ml2]}>
                 {props.children || (
-                    <Text style={[props.isError ? styles.formError : styles.formHelp, styles.mb0]}>
-                        {props.message}
-                    </Text>
+                    <>
+                        {typeof props.message === 'string' ? (
+                            <Text style={textStyles}>
+                                {props.message}
+                            </Text>
+                        ) : _.map(props.message, msg => (
+                            <Text key={msg} style={textStyles}>
```

Comment:
Is the `msg` unique? Two of the same messages will throw a unique "key" prop warning